tests/robot-tests/tests/libs/slack.py

```diff
@@ -0,0 +1,96 @@
+import os
+from slack_sdk import WebClient
+from slack_sdk.errors import SlackApiError
+import shutil
+import json
+import requests
+from bs4 import BeautifulSoup
+
+PATH = f'{os.getcwd()}{os.sep}test-results'
+
+
+def _generate_slack_attachments():
+    failed_tests = dict()
+    passed_tests = dict()
+    skipped_tests = dict()
+
+    report = open(f'{PATH}{os.sep}output.xml', 'rb')
+    contents = report.read()
+
+    soup = BeautifulSoup(contents, 'xml')
+    test = soup.find('total').find('stat')
+
+    failed_tests = test['fail']
+    passed_tests = test['pass']
+    skipped_tests = test['skip']
+
+    return [
+        {
+            "pretext": "All results",
+            "color": "danger" if failed_tests or skipped_tests is not None or not "" else "good",
```

Comment:
Think we could simplify this to: `"danger" if failed_tests or skipped_tests else "good"`

Python has pretty similar boolean coercion rules to JS so things like empty strings, 0 and None are already considered as falsy.

Would suggest going through the other conditionals in this file and apply similar treatment